builtin/credential/ldap/path_users.go

```diff
@@ -20,6 +20,9 @@ func pathUsersList(b *backend) *framework.Path {
 
 		HelpSynopsis:    pathUserHelpSyn,
 		HelpDescription: pathUserHelpDesc,
+		DisplayAttrs: &framework.DisplayAttributes{
+			Navigation: true,
+		},
```

Comment:
@kalafut could the groups field to be a `CommaStringSlice` type?

Reply:
Also it’s likely intended but afaict there’s no validation on whether a group or policy exists when creating LDAP Groups & Users.

Reply:
@madalynrose I think it could be converted. Lemme check

Reply:
https://github.com/hashicorp/vault/pull/6942